kernel_gateway/tests/test_mixins.py

```diff
@@ -4,10 +4,125 @@
 
 import json
 import unittest
+
+try:
+    from unittest.mock import Mock, MagicMock
+except ImportError:
+    # Python 2.7: use backport
+    from mock import Mock, MagicMock
+
 from tornado import web
-from kernel_gateway.mixins import JSONErrorsMixin
+from kernel_gateway.mixins import TokenAuthorizationMixin, JSONErrorsMixin
+
+class SuperTokenAuthHandler(object):
```

Comment:
Is there a reason for defining this class just to have a `prepare()` instead of implementing prepare in `TestableTokenAuthHandler` which is the only subclass?